RFC.md

```diff
@@ -341,6 +341,183 @@ const uploadedObjectReference = await uploadTask.result;
 
 Try out the new `storage` types here: https://www.typescriptlang.org/play#gist/292f4e24178bfca5881aa20961b930dc
 
-# `API` & `Datastore` Changes
+# `API` category Changes
+
+## First param is an object with named parameters
+
+**Amplify v5 (`aws-amplify@5`)**
+
+```typescript
+const apiName = 'MyApiName';
+const path = '/path';
+const myInit = {
+  headers: {}, // OPTIONAL
+  response: true, // OPTIONAL (return the entire Axios response object instead of only response.data)
+  queryStringParameters: {
+    name: 'param' // OPTIONAL
+  }
+};
+
+API.get(apiName, path, myInit)
+  .then((response) => {
+    // Add your code here
+  })
+  .catch((error) => {
+    console.log(error.response);
+  });
+```
+
+```typescript
+API.get({
+    apiName: 'MyApi',
+    path: '/items',
+    authMode: 'AWS_IAM'
+}, myInit)
```

Comment:
Doesn't need to happen now, but why was `myInit` excluded from the param object? We should try to be consistent between category APIs one way or the other.

Reply:
You mean having two arguments?

Reply:
Yeah in this PR it has two arguments but I was wondering if it should just be one object param